pkg/apis/hco/v1alpha1/hyperconverged_resources.go

```diff
@@ -19,23 +21,27 @@ func (r *HyperConverged) getNamespace(defaultNamespace string, opts []string) st
 	return defaultNamespace
 }
 
-func (r *HyperConverged) getLabels() map[string]string {
+func (r *HyperConverged) getLabels(component hcoutil.AppComponent) map[string]string {
 	hcoName := hcoutil.HyperConvergedName
 
 	if r.Name != "" {
 		hcoName = r.Name
 	}
 
 	return map[string]string{
-		hcoutil.AppLabel: hcoName,
+		hcoutil.AppLabel:          hcoName,
```

Comment:
What about the `app.kubernetes.io/instance` instance label, should we be adding that as well?

Reply:
We discussed which labels we need to implement and possible key/value pairs. We didn't invest on this "instance" label at the moment.